### PR TITLE
Prepend the request ID to the output path

### LINKED ISSUE #914: Results should be stored grouped per request
Currently Turbinia stores results in a flat folder structure with a folder per task. It would be a lot more convenient if output was stored in a structure with a folder per request.
Example: https://github.com/google/turbinia/commit/b9bf9dcaac2cbadb217278c77a5b90d4d286760b

### PR DESCRIPTION
**Pull request by dianakramer**
Fix #914

### CODE REVIEW

**Comment by hacktobeer** [comment]
@alimez FYI as you were also adding group id logic to Turbinia.

**Review by hacktobeer — APPROVED** [review]
LGTM, thanks for this!

@alimez can you have a quick look to see if this works for your groupid work? Should be easy to plugin in your (future) PR.

File: turbinia/output_manager.py [hunk 1]
```diff
@@ -52,19 +52,21 @@ def __init__(self):
     self.is_setup = False
 
   @staticmethod
-  def get_output_writers(name, uid, remote_only):
+  def get_output_writers(name, uid, reqid, remote_only):
```
Comment (alimez) [inline turbinia/output_manager.py:55]: For consistency, please change this to `request_id`

Comment (alimez) [reply turbinia/output_manager.py:55]: Also, doesn't it already add the request ID to the path?

**Review by alimez — COMMENTED** [review]
Perhaps we should wait for bulk processing PR to be submitted before submitting this. That way, we can also add the group_id to the path

File: turbinia/output_manager.py (hunk 1 above)
Comment (hacktobeer) [reply turbinia/output_manager.py:55]: Had a look at both code and current output structure and I don't see where it adds request_id? Where do you see it?

**Comment by hacktobeer** [comment]
@alimez I would love to have request_id in there as the output is currently a mess with task_ids at the highest level. What's the ETA on this group_id?
As this is a small and super impactful PR I would love to have it in, makes debugging and getting output of a request a lot easier browsing NFS/GCS.

File: turbinia/output_manager.py (hunk 1 above)
Comment (dianakramer) [reply turbinia/output_manager.py:55]: I cherry picket turbinia-ng. I can modify reqid per request_id. 
In output_manager.py is reqid and in output_manager_test.py is request_id.

Comment (alimez) [reply turbinia/output_manager.py:55]: @hacktobeer I thought it adds request_id in turbiniactl https://github.com/google/turbinia/blob/75b66c6d4f4765d45ee5d7574e1189e1def598c3/turbinia/turbiniactl.py#L762, but I do see it being called with task_id as well and get your point on annoyance.

Comment (alimez) [reply turbinia/output_manager.py:55]: @dianakramer we use `request_id` all over the code. Keeping it consistent will make it more readable and less confusing :)

**Comment by alimez** [comment]
LGTM, please update `reqid` to `request_id`

**Review by alimez — APPROVED** [review]

**Comment by hacktobeer** [comment]
@dianakramer Thank you for this contribution and congratulations on your first Google org PR!

**Pull request merged**